code/tests/functional/backend_api/request_matching.py

```diff
@@ -0,0 +1,81 @@
+from pytest_httpserver import HTTPServer
+from werkzeug.wrappers import Request
+
+
+class RequestMatcher:
```

Comment:
Did you say you opened an issue for pytest_httpserver? Might be useful to link it here, so that the next person knows when this code can be deprecated